src/test/java/com/google/sps/ImageSelectionTest.java

```diff
@@ -0,0 +1,80 @@
+package com.google.sps.image;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class ImageSelectionTest {
+
+  private static final String bingImgLinkStart = "https://tse";
+
+  private static void checkLink(ImageSelection imageSelection) {
+    try {
+      String actualLink = imageSelection.getBestImage();
+      Assert.assertEquals(bingImgLinkStart, actualLink.substring(0, 11));
+    } catch(IOException ex) {
+      StringWriter error = new StringWriter();
```

Comment:
There is a nicer way of doing this:  assertThat(ex.getMessage(), is("the expected result"));